gtfs/importers/gtfs_feed_importer.py

```diff
@@ -0,0 +1,278 @@
+                    gtfs_value, model_field, gtfs_field
+                )
+            if model == CalendarDate and creation_attributes["calendar_id"] is None:
+                # We don't support calendar dates without a calendar at least for now
+                num_of_skipped += 1
+            else:
+                objs_to_create.append(
+                    model(
+                        feed_id=feed.id,
+                        **creation_attributes,
+                    )
+                )
+
+            if (
+                num_of_processed % self.object_creation_batch_size == 0
+                or num_of_processed >= num_of_rows  # yes this should never be GT
+            ):
+                created_objs = model.objects.bulk_create(objs_to_create)
+
+                # update ID cache
+                if "source_id" in creation_attributes:
+                    self.id_cache[model].update(
+                        {o.source_id: o.id for o in created_objs}
+                    )
+
+                objs_to_create = []
+
+                self.logger.debug(
+                    f"Processed {num_of_processed}/{num_of_rows} {plural_name}"
+                )
+                if num_of_processed >= num_of_rows and num_of_skipped:
+                    self.logger.info(f"Skipped {num_of_skipped} {plural_name}")
+
+    def _convert_value(self, gtfs_value, model_field, gtfs_field):
+        if isinstance(model_field, models.ForeignKey):
+            value = self._get_related_obj_id(model_field.related_model, gtfs_value)
+            # empty agency_id should default to the only agency there (hopefully) is
+            # in the feed
+            if gtfs_field == "agency_id" and not value:
+                value = self._get_default_agency_id()
+            return value
+        elif isinstance(model_field, models.TimeField):
+            return self._convert_time(gtfs_value)
+        elif isinstance(model_field, models.DateField):
+            return self._convert_date(gtfs_value)
+        elif isinstance(model_field, models.PointField):
+            return self._convert_point(gtfs_value)
+        elif isinstance(model_field, models.IntegerField):
+            return self._convert_int(gtfs_value)
+        else:
+            # no conversion needed
+            return gtfs_value
+
+    def _get_related_obj_id(self, related_model, source_id):
+        return self.id_cache[related_model].get(source_id)
+
+    def _get_default_agency_id(self):
+        agency_ids = self.id_cache[Agency].values()
+        if len(agency_ids) != 1:
+            raise GTFSFeedImporterError(
+                f"Something fishy going on! Trying to use default agency but there "
+                f"are {len(agency_ids)}) agencies in the feed."
+            )
+        return next(iter(agency_ids))
+
+    @staticmethod
+    def _convert_date(gtfs_date_str):
+        return datetime.strptime(gtfs_date_str, "%Y%m%d").date()
+
+    @staticmethod
+    def _convert_time(gtfs_time_str):
```

Comment:
As discussed IRL we might be able to solve this issue with Django's built-in [DurationField](https://docs.djangoproject.com/en/dev/ref/models/fields/#durationfield). But that can surely be refactored and thought later.